README_developers.md

```diff
@@ -125,9 +141,13 @@ RAJA::forall<hiop_raja_exec>(
   RAJA_LAMBDA(RAJA::Index_type i)
   {
     if(svec[i]==1.)
+    {
       local_data_dev[i]=c;
```

Comment:
```suggestion
      local_data_dev[i] = c;
```
I would suggest leaving spaces around operators. Notable exceptions being member access, address-of, and dereferencing operators `.`, `&`, `*` and `->`.